### PR TITLE
Add reactive testnet

### PR DESCRIPTION
**Pull request by Zorato**
### Description

Add Kopli Reactive (testnet) chain. EVM compatible.

### CODE REVIEW

File: chains/reactivekopli/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://kopli-api.reactscan.net/
```
Comment (JaunePomme) [inline chains/reactivekopli/metadata.yaml:3]: take the last "/" off I think

File: chains/reactivekopli/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    family: blockscout
+    name: reactscan
+    url: https://kopli.reactscan.net/
```
Comment (JaunePomme) [inline chains/reactivekopli/metadata.yaml:6]: Same, take the last "/" off I think

**Comment by JaunePomme** [comment]
Modify your metadata.yml a bit
Could add the deployer name in the metadata.yml
Run prettier
Minimify the svg, here is the link : https://jakearchibald.github.io/svgomg/
And check other PR cuz i'm not sure you need the height and width in it.
Then I think it's good !

**Comment by daniil-parsiq** [comment]
Thanks for the review, I fixed everything according to your comments.

File: chains/reactivekopli/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://kopli-api.reactscan.net
+    family: blockscout
```
Comment (paulbalaji) [inline chains/reactivekopli/metadata.yaml:4]:
```suggestion
    family: other
```

File: chains/reactivekopli/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://kopli-api.reactscan.net
```
Comment (paulbalaji) [inline chains/reactivekopli/metadata.yaml:3]: are there any docs r.e. the reactscan explorer API

Comment (daniil-parsiq) [reply chains/reactivekopli/metadata.yaml:3]: well... there really isn't. but we are ready to add necessary methods on a short notice, question is - where we can find some sort of definition for required methods

Comment (paulbalaji) [reply chains/reactivekopli/metadata.yaml:3]: typically most explorers will have some endpoint that is compatible with the etherscan api spec https://docs.etherscan.io/api-endpoints/contracts#verify-solidity-source-code

e.g. blockscout docs point here https://docs.blockscout.com/devs/apis/rpc

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**